.travis-xs-opam.sh

```diff
@@ -1,34 +1,14 @@
 #!/bin/bash
-# SUMMARY:
-# Builds & tests xapi with coverage in a Ubuntu 16.04 Docker container with
-# OCaml 4.04.2, then uploads the coverage information to coveralls.
 
 set -ex
 
-# Currently there is no way of specifying OPAM depexts for multiple versions of
-# a given disto, and our current depexts only work with Ubuntu >= 16.04, due to
-# a change in packages (libsystemd-dev). Since the build environments of Travis
-# are older then Ubuntu 16.04, we have to run the build in a Docker container
-# with an appropriate Ubuntu version.
-# We need to pass some Travis environment variables to the container to enable
-# uploading to coveralls and detection of Travis CI.
-docker run --rm --volume=$PWD:/mnt --workdir=/mnt \
-  --env "TRAVIS=$TRAVIS" \
-  --env "TRAVIS_JOB_ID=$TRAVIS_JOB_ID" \
-  ocaml/opam2:ubuntu-18.04-ocaml-4.07 \
-  bash -ex -c '
-sudo apt-get update
+PACKAGE="xapi"
+PINS="xapi:."
+BASE_REMOTE="https://github.com/xapi-project/xs-opam.git"
+DISTRO="debian-9-ocaml-4.07"
+TRAVIS="$TRAVIS"
+TRAVIS_JOB_ID="$TRAVIS_JOB_ID"
```

Comment:
Are the the two travis env variables needed?

Reply:
I don't know - I just kept them. But they might have depend on the Dockerfile that we are no longer using.

Reply:
I don't think so, they were needed for uploading coverage to coveralls